src/methods/receiver.md

```diff
@@ -13,3 +13,5 @@ are other possible receivers for a method:
   transmitted.
 * No receiver: this becomes a static method on the struct. Typically used to
   create constructors which are called `new` by convention.
+
+There are even more types, e.g., `Box<Self>`.
```

Comment:
Should we have a discussion of why you'd want to use alternate receiver types rather than relying on `impl Deref`, or do we want to rely on the instructor for that?

Reply:
I think this is something we'll have to defer to the speaker notes (i.e., to the instructor as you suggest).

As you hint, the pages themselves cannot go into detail on all topics — plus, people are seeing this on Day 2 after having been exposed to Rust for 5-6 hours. I think we should stick to the main point: how does a method look in Rust and related and when do you need to use `&mut self` instead of just `&self`.